docs/user/monitoring/kibana-alerts.asciidoc

```diff
@@ -58,6 +58,71 @@ monitoring data. By default, the trigger condition is set to missing for 15 minu
 looking back 1 day. The alert is grouped across all the nodes of the cluster by running
 checks on a schedule time of 1 minute with a re-notify internal of 6 hours. 
 
+[discrete]
+[[kibana-alerts-thread-pool-rejections]]
+== Thread pool rejections (search/write)
+
+This alert is triggered when a node is experience thread pool rejections. By
+default, the trigger condition is set at 300 or more over the last 5
+minutes. The alert is grouped across all the nodes of the cluster by running
+checks on a schedule time of 1 minute with a re-notify internal of 1 day. 
+Thresholds can be set independently for `search` and `write` type rejections.
+
+[discrete]
+[[kibana-alerts-ccr-read-exceptions]]
+== CCR read exceptions
+
+This alert is triggered if a read exception has been detected on any of the 
+replicated clusters. The trigger condition is met if 1 or more read exceptions 
+are detected in the last hour. The alert is grouped across all replicated clusters 
+by running checks on a schedule time of 1 minute with a re-notify internal of 6 hours. 
+
+[discrete]
+[[kibana-alerts-large-shard-size]]
+== Large shard size
+
+This alert is triggered if a large (primary) shard size is found on any of the 
+specified index patterns. The trigger condition is met if an index's shard size is 
+55gb or over in the last 5 minutes. The alert is grouped across all indices that match 
+the default patter of `*` by running checks on a schedule time of 1 minute with a re-notify 
+internal of 12 hours.
+
+[discrete]
+[[kibana-alerts-cluster-alerts]]
+== Cluster alerts
+
+These alerts summarize the current status of your {stack}. You can drill down into the metrics 
+to view more information about your cluster and specific nodes, instances, and indices.
+
+An alert will be triggered if any of the following conditions are met within the last minute:
+
+* {es} Cluster Health Status is Yellow (missing at least one replica)
+or Red (missing at least one primary).
+* {es} Version Mismatch. You have {es} nodes with
+different versions in the same cluster.
+* {kib} Version Mismatch. You have {kib} instances with different
+versions running against the same {es} cluster.
+* Logstash Version Mismatch. You have Logstash nodes with different
+versions reporting stats to the same monitoring cluster.
+* {es} Nodes Changed. You have {es} nodes that were recently added or removed.
+* {es} License Expiration. The cluster's license is about to expire.
++
+--
+If you do not preserve the data directory when upgrading a {kib} or
+Logstash node, the instance is assigned a new persistent UUID and shows up
+as a new instance
+--
+* {xpack} License Expiration. When the {xpack} license expiration date
```

Comment:
> {xpack} license expiration

... is there still such a thing as an X-Pack license? Or should we be referring to subscriptions?